nautobot/dcim/choices.py

```diff
@@ -1117,14 +1312,21 @@ class CableStatusChoices(ChoiceSet):
 
 class CableLengthUnitChoices(ChoiceSet):
 
+    # Metric
+    UNIT_KILOMETER = "km"
     UNIT_METER = "m"
     UNIT_CENTIMETER = "cm"
+
+    # Imperial
+    UNIT_MILE = "mi"
```

Comment:
These need corresponding logic in the `to_meters` utility function to handle these units - see https://github.com/netbox-community/netbox/commit/a6d937b366931670d8cc19b5dd6564f2efee3d85 and https://github.com/netbox-community/netbox/blob/develop/netbox/utilities/utils.py#L276 for reference.

Reply:
good catch! I've added it